packages/gappa/gappa.1.4.0/opam

```diff
@@ -15,6 +15,8 @@ build: [
   [ "./configure"
     # If someone knows how to ask MacPorts for the usual include and lib path, please tell me
     "CXXFLAGS=-I/opt/local/include" "LDFLAGS=-L/opt/local/lib" { os-distribution = "macports" & os = "macos" }
+    # Support installing on Apple Silicon with Homebrew
+    "CXXFLAGS=-I/opt/homebrew/include" "LDFLAGS=-L/opt/homebrew/lib" { os-distribution = "homebrew" & os = "macos" }
```

Comment:
```suggestion
    "CXXFLAGS=-I/opt/local/include" "LDFLAGS=-L/opt/local/lib" { os-distribution = "macports" & os = "macos" & arch = "x86_64" }
    # Support installing on Apple Silicon with Homebrew
    "CXXFLAGS=-I/opt/homebrew/include" "LDFLAGS=-L/opt/homebrew/lib" { os-distribution = "homebrew" & os = "macos" & arch = "arm64" }
```
Otherwise it both would always be set and I’m not sure both would be concatenated. I think they would be always overwritten

Reply:
Thanks for the suggestion. I was under the impression that `os-distribution` value might distinguish these, but arch is certainly helpful and descriptive.

Reply:
oh.. Nevermind I read it wrong. I thought it was homebrew twice. Could you remove the last commit?

Reply:
Ah, done.

Reply:
Wait, I was too fast at reading the change.
Wouldn't this make `"CXXFLAGS=-I/opt/local/include"` always set and always be overridden by `"CXXFLAGS=-I/opt/local/include"`? Am I mistaken that the filter only applies to the LDFLAGS string?